Providers/Modules/Plugins/Common/plugin/out_oms_changetracking_file.rb

```diff
@@ -0,0 +1,530 @@
+module Fluent
+
+  class OutChangetrackingFile < BufferedOutput
+
+    Plugin.register_output('out_oms_oms.changetracking.file', self)
+	
+    # Endpoint URL ex. localhost.local/api/
+
+    def initialize
+      super
+	  
+      require 'base64'
+      require 'digest'
+      require 'json'
+      require 'net/http'
+      require 'net/https'
+      require 'openssl'
+      require 'rexml/document'
+      require 'securerandom'
+      require 'socket'
+      require 'uri'
+      require_relative 'omslog'
+      require_relative 'oms_configuration'
+      require_relative 'oms_common'
+    end
+
+    config_param :omsadmin_conf_path, :string, :default => '/etc/opt/microsoft/omsagent/conf/omsadmin.conf'
+    config_param :cert_path, :string, :default => '/etc/opt/microsoft/omsagent/certs/oms.crt'
+    config_param :key_path, :string, :default => '/etc/opt/microsoft/omsagent/certs/oms.key'
```

Comment:
I think his reference was the original out_oms file which has the constants in the same way -
 https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/source/code/plugins/out_oms.rb#L21-L25